bootstrap/v2/pkg/kfapp/existing_arrikto/existing.go

```diff
@@ -154,13 +155,23 @@ func (existing *Existing) Apply(resources kftypes.ResourceEnum) error {
 		KubeflowEndpoint        string
 		OIDCEndpoint            string
 		AuthServiceClientSecret string
+		AuthServiceHmacSecret   string
+		OIDCRedirectUris        []string
```

Comment:
Maybe we should call this OIDCExtraRedirectURIs since the default redirect uri is included no matter what.